webapp/channels/src/components/admin_console/data_retention_settings/index.ts

```diff
@@ -28,10 +29,14 @@ type Actions = {
 function mapStateToProps(state: GlobalState) {
     const customPolicies = getDataRetentionCustomPolicies(state);
     const customPoliciesCount = getDataRetentionCustomPoliciesCount(state);
+    const globalMessageRetentionHours = getDataRetentionTimeInHours(state.entities.admin.config.DataRetentionSettings?.MessageRetentionDays, state.entities.admin.config.DataRetentionSettings?.MessageRetentionHours);
+    const globalFileRetentionHours = getDataRetentionTimeInHours(state.entities.admin.config.DataRetentionSettings?.MessageRetentionDays, state.entities.admin.config.DataRetentionSettings?.MessageRetentionHours);
```

Comment:
Same here, You used the above `message` value into the file variable. `MessageRetentionDays` and `MessageRetentionHours`.